README.md

```diff
@@ -138,6 +138,35 @@ PublicReportingAPI provides the corresponding methods for reporting logs into th
 * launchError (*message* , *file*). Reports *message* and optional *file* as a log of the launch with error log level.
 * launchFatal (*message* , *file*). Reports *message* and optional *file* as a log of the launch with fatal log level.
 
+#### Finish launch/suite/spec with status
+PublicReportingAPI provides the following methods for setting status into the current suite/spec.
+
+* setStatus(*status*, *suite*). Assign *status* to the current suite/spec.<br/>
+*status* should be equal to one of the following values: *passed*, *failed*, *stopped*, *skipped*, *interrupted*, *cancelled*, *info*, *warn*.<br/>
+*suite* it's description of your suite (all suite descriptions must be unique) ***REQUIRED INSIDE OF YOUR SUITE, OPTIONAL FOR SPEC*** <br/>
+
+* setStatusPassed(). Assign *passed* status to the current suite/spec.
+* setStatusFailed(). Assign *failed* status to the current suite/spec.
+* setStatusSkipped(). Assign *skipped* status to the current suite/spec.
+* setStatusStopped(). Assign *stopped* status to the current suite/spec.
+* setStatusInterrupted(). Assign *interrupted* status to the current suite/spec.
+* setStatusCancelled(). Assign *cancelled* status to the current suite/spec.
+* setStatusInfo(). Assign *info* status to the current suite/spec.
+* setStatusWarn(). Assign *warn* status to the current suite/spec.
+
+PublicReportingAPI provides the corresponding methods for setting status into the launch.
+* setLaunchStatus(*status*). Assign *status* to the launch.<br/>
+*status* should be equal to one of the following values: *passed*, *failed*, *stopped*, *skipped*, *interrupted*, *cancelled*, *info*, *warn*.<br/>
+
+* setLaunchStatusPassed(). Assign *passed* status to the launch.
+* setLaunchStatusFailed(). Assign *failed* status to the launch.
+* setLaunchStatusSkipped(). Assign *skipped* status to the launch.
+* setLaunchStatusStopped(). Assign *stopped* status to the launch.
+* setLaunchStatusInterrupted(). Assign *interrupted* status to the launch.
+* setLaunchStatusCancelled(). Assign *cancelled* status to the launch.
+* setLaunchStatusInfo(). Assign *info* status to the launch.
```

Comment:
Does the launch supports `info` and `warn` statuses?

Reply:
According to the answer from the backend of the developer, the statuses for the suite and step are also suitable for the launch

Reply:
Got it, thank you